Sources/Plasma/CoreLib/HeadSpin.h

```diff
@@ -209,7 +213,9 @@ inline double hsSwapEndianDouble(double dvalue)
 // Generic versions for use in templates
 template <typename T> inline T hsToLE(T value) = delete;
 template <> inline char hsToLE(char value) { return value; }
+#ifndef _CHAR_IS_SIGNED
 template <> inline int8_t hsToLE(int8_t value) { return value; }
+#endif
```

Comment:
This probably isn't actually the best fix for this (after all, the difference is that `int8_t` is defined as `char` rather than `signed char`, not that `char` is signed).  Instead, we should probably rewrite these to use `signed char` and `unsigned char` instead of `int8_t` and `uint8_t`, which also matches the fixes we made to libhsplasma.